core/src/test/java/org/springframework/security/authentication/UserDetailsRepositoryReactiveAuthenticationManagerTests.java

```diff
@@ -171,4 +172,90 @@ public void authenticateWhenPostAuthenticationChecksNotSet() {
 
 		verifyZeroInteractions(this.postAuthenticationChecks);
 	}
+
+	@Test
+	public void testAuthenticateFailsIfAccountExpired() {
```

Comment:
Please update the test names to have the form _methodWhenConditionThenResult_. This one could be `authenticateWhenAccountExpiredThenException`.